### DISCUSSION TITLE
Fluentd plugin to connect to splunk

### DISCUSSION THREAD

**Discussion by mrudrego** [fluent/fluentd#4312] [Q&A] [answered]
Hi,

Was looking for opensource fluentd plugin which connects to splunk without any licensing/enterprise version. Came across this [fluent-plugin-splunk](https://github.com/fluent/fluent-plugin-splunk) , but the installation command of the gem says "fluent-gem install fluent-plugin-splunk-**enterprise**"
Does this mean this plugin can be used only for enterprise version of splunk or can it be used even for free version of splunk without any licensing issues?

Thanks in advance,

**Comment by cosmo0920** [comment, marked as answer]
This is because there is a naming glitches for its repo and actual gem name.
The repo name is [fluent-plugin-splunk](https://github.com/fluent/fluent-plugin-splunk). However, fluent-plugin-splunk is already registered in rubygems and ought to avoid already registered name to ensure uniqueness of the gem name.
So, we use "fluent-gem install fluent-plugin-splunk-**enterprise**", instead.

**Comment by mrudrego** [comment]
Hi @cosmo0920 , is this [fluent-plugin-splunk](https://github.com/fluent/fluent-plugin-splunk) plugin still supported ? The last commit on this repo seems to be around 4 years ago. So wanted to check if this is still supported and not EOL.

Thanks in advance,

**Reply by cosmo0920** [reply]
Ping @daipom or @kenhys

**Comment by mrudrego** [comment]
@daipom  @kenhys  , can you please help in knowing if https://github.com/fluent/fluent-plugin-splunk is still supported ?

Thanks,

**Reply by daipom** [reply]
@mrudrego Sorry for my late response.
It's still supported (It’s not that we’ve stopped developing it), but I'm not sure how well it actually works.
It seems to be using an old API, and there's no CI. It might need more development.

It seems that it was not developed much because other plugins, such as [fluent-plugin-splunk-hec](https://github.com/splunk/fluent-plugin-splunk-hec), were more commonly used. (However, `fluent-plugin-splunk-hec` is EOL now)

**Reply by cosmo0920** [reply]
Yeah, but Splunk documentation says that /services/collector/event and /services/collector/event/raw are not deprecated. So, I supposed that it could be work for the current splunk:
https://docs.splunk.com/Documentation/Splunk/9.4.2/RESTREF/RESTinput#services.2Fcollector
https://docs.splunk.com/Documentation/Splunk/9.4.2/RESTREF/RESTinput#services.2Fcollector.2Fraw
